Assets/Scripts/UI/DialogBox/Settings/New Settings/SettingsCategory.cs

```diff
@@ -0,0 +1,114 @@
+﻿#region License
+// ====================================================
+// Project Porcupine Copyright(C) 2016 Team Porcupine
+// This program comes with ABSOLUTELY NO WARRANTY; This is free software, 
+// and you are welcome to redistribute it under certain conditions; See 
+// file LICENSE, which is part of this source code package, for details.
+// ====================================================
+#endregion
+using System.Xml;
+using System.Collections.Generic;
+
+/// <summary>
+/// Holds the category and its options (and name).
+/// No need to write, only will ever read.
```

Comment:
Yeh, I normally don't focus on writing proper english till later (when I do cleanups), but I'll make the changes regardless :D.